extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -68,6 +70,29 @@ amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-s
   background: #EDEDED !important;
 }
 
+.i-amphtml-story-subscriptions-dialog-banner-button {
+  height: 100% !important;
+  display: flex !important;
+  justify-content: flex-end !important;
+  align-items: center !important;
+  letter-spacing: 0.015em !important;
+  margin-right: 0.58em !important;
+  font-size: 0.75em !important;
+  opacity: 0 !important;
+  transition: opacity 0.3s cubic-bezier(0.42, 0.0, 1, 1) !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner-button.i-amphtml-story-subscriptions-dialog-banner-button-visible {
+  opacity: 1 !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner-button-logo {
+  width: 1.4em !important;
+  height: 1.4em !important;
+  background-size: contain !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"><path d="M9.00002 6.71075C8.61002 7.10075 8.61002 7.73075 9.00002 8.12075L12.88 12.0008L9.00002 15.8808C8.61002 16.2708 8.61002 16.9008 9.00002 17.2908C9.39002 17.6808 10.02 17.6808 10.41 17.2908L15 12.7008C15.39 12.3108 15.39 11.6808 15 11.2908L10.41 6.70075C10.03 6.32075 9.39002 6.32075 9.00002 6.71075Z" fill="black"/></svg>') !important;
```

Comment:
Switch out the svg for this pls:
```
<svg xmlns="http://www.w3.org/2000/svg" width="8" height="12" stroke="black" fill="none" stroke-width="2px" stroke-linecap="round" stroke-linejoin="round"><polyline points="1.75,1.5 6,6 1.75,10.6"/></svg>
```

This is a polyline with a stroke instead of the points that outline the path.
If interested, the [SVG drawing API](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Basic_Shapes) is super cool.

Reply:
We also need the SVG to flip when it's in RTL.
Using `transform: scale(-1, 1)` is a trick to reflect things vertically in the dom.

Reply:
Done. what's the tool you use to convert this? The one from Matias doc: https://jakearchibald.github.io/svgomg/?